tests/unit/test_intercept.py

```diff
@@ -30,6 +30,29 @@ def test_read_write_exec_trace(self):
             result = sut.parse_exec_trace(temp_file)
             self.assertEqual(input_one, result)
 
+    def test_read_write_exec_trace_with_response(self):
+        with libear.temporary_directory() as tmp_dir:
+            response_file_one = os.path.join(tmp_dir, 'response1.jom')
+            response_file_two = os.path.join(tmp_dir, 'response2.jom')
+            input_one = Execution(
+                pid=123,
+                cwd='/path/to/here',
+                cmd=['cc', '-c', '@'+response_file_one, 'this.c',
+                     '@'+response_file_two])
+            output_one = Execution(
+                pid=123,
+                cwd='/path/to/here',
+                cmd=['cc', '-c', 'that.c', 'this.c', '-Idoes_not_exists'])
```

Comment:
I don't think it's a valid compiler call. You can't specify `-c` and two source files at the same time. (I know it's just a bunch of string in a text file, but tests are examples which helps people to understand what's going on.)